lib/loaders/active_record_loader.rb

```diff
@@ -19,20 +19,23 @@ def initialize(params)
         @data = params[:mable_etl_data]
         @silence_log_config = params[:silence_log]
         @logger = params[:logger]
+        @inserted_count = 0
       end
 
       def load
         slience_log do
-          ActiveRecord::Base.transaction do
-            @active_record_model_name.insert_all(@data)
+          @data.in_groups_of(10_000) do |group|
```

Comment:
whats the reason for giving the option to silence the log here?

Reply:
We thought it may have been causing an error due to it's size however it may have been the query trying to refetch that was causing an error